dali/sasha/saarch.cpp

```diff
@@ -83,297 +83,446 @@ static void mkDateCompare(bool dfu,const char *dt,StringBuffer &out,char fill)
     }
 }
 
+class CWUData : public CInterface
+{
+public:
+    CDateTime modifiedTime;
+    Owned<IPropertyTree> wuTree;
+};
+
+static CriticalSection wuDataSortSect;
+static bool wuDataSortInc = true;
+static IArrayOf<IPropertyTree> *wusRef = nullptr;
```

Comment:
These three global/statics should be removed.

Reply:
ok